### ISSUE TITLE
V0.15.0 Missing dependency: source-map-support

### ISSUE THREAD

**Issue by Rachnerd** [dotansimha/graphql-code-generator#1029]
**Describe the bug**
Upgrading to V0.15.0 breaks due to a missing dependency `source-map-support`.

**To Reproduce**
I've isolated the issue in a new repo that followed the steps on the `installation` chapter on the gql-gen website.

1. clone https://github.com/Rachnerd/graphql-code-generator-issue
2. npm i && npm run generate

**Expected behavior**
Should work without having to manually install `source-map-support`. 

**Environment:**
 - OS: Ubuntu 18.04
 - Codegen: 0.15.0
 - Node: v10.8.0

**Additional context**
Everything works fine after adding the `source-map-support` dep.

**Comment by kamilkisiela** [comment]
I guess it comes from `graphql-tag-pluck` at least that was the case few versions ago.

**Comment by Rachnerd** [comment]
Yes you're right.
```
Error: Cannot find module 'source-map-support'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/rachnerd/Workspace/graphql-code-gen-issue/node_modules/graphql-tag-pluck/build/graphql-tag-pluck.js:1:63)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

**Comment by dotansimha** [comment]
Fixed in 0.15.1 : )
